lib/generators/templates/devise-security.rb

```diff
@@ -41,4 +41,10 @@
 
   # Time period for account expiry from last_activity_at
   # config.expire_after = 90.days
+
+  # Allow passwords to be equal to email (false, true)
+  # This specifically performs case insensitive comparisons
+  # Used by secure_validatable. Defaults to false
+  # config.allow_passwords_equal_to_email = false # deny passwords equal (case insensitive) to the email
```

Comment:
Let's only have one line, the user can change the value as they want (similar to the other configs in this file).

Reply:
Sounds good. Thanks for looking. I'll push that up in a few minutes